scripts/docker_install.sh

```diff
@@ -35,6 +35,14 @@ apt-get install -y --no-install-recommends \
     dnsutils \
     gcc-mingw-w64-x86-64
 
+# Deps required by Qt / linuxdeployqt:
```

Comment:
I thought these would be listed in docker-qt5base/Dockerfile?

Reply:
I am also confused by that, but they have to be in the image that builds the Qt app, not the Qt5 Dockefile (which is dropped completely except for the actual Qt directory that it produces)